Code/GraphMol/DetermineBonds/catch_tests.cpp

```diff
@@ -339,4 +340,17 @@ H	  20.371657	   2.180532	   1.492305
     determineBonds(*m);
     CHECK(m->getNumAtoms() == 102);
   }
-}
\ No newline at end of file
+}
+
+/* TEST_CASE("Github #6121: Single Atom in DetermineBonds") {
+  SECTION("as reported") {
+    std::string xyz = R"XYZ(1
+
+H   0.0         0.0           0.0
+)XYZ";
+    std::unique_ptr<RWMol> m(XYZBlockToMol(xyz));
+    REQUIRE(m);
+    CHECK(determineBonds(*m));
```

Comment:
The reason this won't compile is that `determineBonds()` doesn't return a value, so there's nothing to CHECK.
Given that the problem is an exception thrown by `determineBonds()` and that the exception should no longer be thrown after your fix has been made, you can test that the fix works by just calling the function:
```suggestion
    // before the fix, this would thrown an exception:
    determineBonds(*m);
```

Reply:
Make sense, thank you for the explanation!